packages/demo/ios/Podfile

```diff
@@ -14,6 +14,8 @@ target 'ExampleApp' do
   permissions_path = '../../../node_modules/react-native-permissions/ios'
   pod 'RNPermissions', :path => '../../../node_modules/react-native-permissions'
   pod 'Permission-BluetoothPeripheral', :path => "#{permissions_path}/BluetoothPeripheral"
+  pod 'Permission-LocationAlways', :path => "#{permissions_path}/LocationAlways"
+  pod 'Permission-LocationWhenInUse', :path => "#{permissions_path}/LocationWhenInUse"
```

Comment:
Existe-t-il une librairie pour la permission Motion and Fitness (qu'on appelle Activity nous) ? Il est nécessaire de demander cet accès en même temps que la position

Reply:
Yes react-native-permissions gère la permissions `MOTION`.

Mais on est d'accord qu'on ne doit pas le gérer et que c'est directement géré par le SDK.

En tout cas c'est que j'ai compris avec cette doc :
https://docs.drivequant.com/trip-analysis/ios/get-started#motion-and-fitness-permission